src/models/data.h

```diff
@@ -804,6 +807,13 @@ struct ZoomAction
         return time.isValid();
     }
 };
+
+struct TracepointTimeMeasurementsParameters
+{
+    QRegularExpression startRegex;
+    QString stopRegex;
```

Comment:
this isn't a regex, it's actually a kind of replacement that is fed into the start regex. can you please add a comment that explains the relation and use better names? because clearly stopRegex isn't a regex, or?